### PR TITLE
kubernetes-cli 1.17.0

### PR DESCRIPTION
**Pull request by translucens**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by translucens** [comment]
I confirmed #47630 resolves the build error above.

**Comment by chenrui333** [comment]
Rebased the branch after the `helmsman` fix.

File: Formula/kubernetes-cli.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@ class KubernetesCli < Formula
     sha256 "a97a579a2adaa9720aa122355b0a43cbebea49da394f3489a300f41d8d5e7f77" => :high_sierra
   end
 
-  depends_on "go@1.12" => :build
```
Comment (chenrui333) [inline Formula/kubernetes-cli.rb]: Latest k8s is using go 1.13, relates to https://github.com/kubernetes/kubernetes/pull/82809

**Comment by chenrui333** [comment]
@BrewTestBot test this please

**Comment by dawidd6** [comment]
@BrewTestBot test this please

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @translucens!!

**Pull request closed without merging**